### PR TITLE
Various Mining Tweaks

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Makes some various mining tweaks largely inspired from TG.

ports bluespace crystal portion of: https://github.com/tgstation/tgstation/pull/19062

Changes form that PR: uses recipe datum as opposed to snowflaked attack_hand.

:cl: Fox McCloud
tweak: Removes most of the miner's uniform equipment from their lockers and puts it in a miner's wardrobe
tweak: Miners start off with a lot of their clothing equipment equipped to them
add: Miners now have unique headset sprites
tweak: Miners no longer have lanterns, but a seclite; they can attach this to their kinetic accelerators now
tweak: Miners have workboots instead of black shoes
tweak: Miners no longer have a default mining scanner but an advanced one that works at reduced range (5 instead of 7) and has a longer cooldown between pulses (5 instead of 3.5)
tweak: removed extra mining hardsuit from the outpost
tweak: removed medikit spam from outpost; now just a toxin kit and a regular medical kit
add: Bluespace crystals now earn you points and can be refined into bluespace polycrystal sheets
tweak: Spare vouchers from QM and HoP removed
tweak: removed armor from the fake tactical turtleneck
/:cl:

Pics of radio headset:

img

### CODE REVIEW

**Comment by TullyBurnalot** [comment]
@Fox-McCloud Toxin kit is useless for miners in comparison to Brute/Burn kits.

File: code/modules/projectiles/guns/energy/special.dm [hunk 1]
```diff
@@ -86,20 +86,49 @@
...
+		only throwing more money in the bin. \n\
+		Whenever the Nanotrasen scientists were on the edge of giving up, a \
+		breakthrough was made by head researcher Miles Parks McCollum, who \
+		demonstrated that the cells could be stabilized when exposed to radium \
+		then cooled with cryostylane. After this discovery, the low pressure gun, \
+		now named the Kinetic Accelerator, was hastily completed and made compatible \
+		with the self-recharging cells. As a result of poor testing, the currently \
+		used guns lose their charge when not in use, and when two Kinetic Accelerators \
+		come in proximity of one another, they will interfere with each other. Despite \
+		this, the shoddy guns still see use in the mining of plasma to this day."
```
Comment (Fox-McCloud) [inline code/modules/projectiles/guns/energy/special.dm]: Best item description in the entire game, by far.

**Comment by Taaketa** [comment]
Sad about the spare brute and burn kits :( I always grab the medkit from cargo and then remove the salic acid and injector and add my extra brute patch and one burn patch from the burn kit.

File: code/modules/projectiles/guns/energy/special.dm (hunk 1 above)
Comment (Tastyfish) [inline code/modules/projectiles/guns/energy/special.dm]: :eyes:

**Comment by Fox-McCloud** [comment]
Added in the ability for mining to finally be able to process bluespace crystals in the ORM; they smelt into polycrystal sheets which break down back into bluespace crystals.

**Comment by Taaketa** [comment]
Could you add Burn kits to be point purchasable to the miner vending machines?

File: code/modules/telesci/bscrystal.dm [hunk 2]
```diff
@@ -38,4 +39,22 @@
...
+
+var/global/list/datum/stack_recipe/bluespace_crystal_recipes = list(new/datum/stack_recipe("Breadkdown into bluespace crystal", /obj/item/weapon/ore/bluespace_crystal, 1, one_per_turf = 0, on_floor = 1))
+
+/obj/item/stack/sheet/bluespace_crystal
+	name = "bluespace polycrystal"
+	icon = 'icons/obj/telescience.dmi'
+	icon_state = "polycrystal"
+	desc = "A stable polycrystal, made of fused-together bluespace crystals. You could probably break one off."
+	origin_tech = "bluespace=4;materials=3"
+	attack_verb = list("bluespace polybashed", "bluespace polybattered", "bluespace polybludgeoned", "bluespace polythrashed", "bluespace polysmashed")
```
Comment (KasparoVy) [inline code/modules/telesci/bscrystal.dm]: Lmfao, excellent

**Comment by FlattestGuitar** [comment]
Survival medipens when.

**Comment by Vivalas** [comment]
I really don't like replacing lanterns with seclites, but that's me. It feels out of place.

**Comment by TullyBurnalot** [comment]
I once again question the logic of giving miners a _Toxins_ FAK.

Miners are going to be handling mostly Burn or Brute damage. Why give them a Toxins kit and not a much more relevant Brute/Burn?

**Comment by Fox-McCloud** [comment]
>Miners are going to be handling mostly Burn or Brute damage. Why give them a Toxins kit and not a much more relevant Brute/Burn?

The sleeper change will handle the latter without making them able to sustain forever with just starting equipment.

**Comment by LittleBigKid2000** [comment]
Make regular flashlights and/or lanterns able to be mounted to kinetic accelerators.

**Comment by Taaketa** [comment]
@Fox-McCloud Ability to buy burn kits from miner vending machine for points would be nice.

**Comment by Spacemanspark** [comment]
What Taaketa said, but make it expensive.

**Comment by Davdichan** [comment]
Burn kits would be a godsend, or just up the point cost for the brute kit by 50% and make half the items in it heal burns, I.E. 2 silver sulfide patches, 2 styptic patches, 2 saline injectors and the health analyzer.

That or making a 1000 point synth flesh kit

**Pull request merged**